src/test/java/com/vaccin/vaccin/VaccinApplicationTests.java

```diff
@@ -31,25 +29,78 @@
 
 	@Before
 	public void setUp() {
-		VaccineCenter vc = new VaccineCenter();
-		vc.setAddress("O adresa");
-		vc.setLongitude(0D);
-		vc.setLatitude(0D);
-		VaccineType vt = new VaccineType();
-		vt.setBrand("Some brand");
-		vc.setVaccineType(vt);
-		var lst = new ArrayList<>();
-		lst.add(vc);
 		Mockito.when(vaccineCenterRepository.findAll())
-				.thenReturn((List) lst);
+				.thenReturn((List) GenerateCenters.getCenters(0L,6));
+
+		Mockito.when(vaccineTypeRepository.findById(0L))
+				.thenReturn(Optional.empty());
+
+		VaccineType vt = new VaccineType();
+		vt.setId(1L);
+		vt.setBrand("Some brand 1");
+		vt.setDaysBetweenShots(60);
+		Mockito.when(vaccineTypeRepository.findById(1L))
+				.thenReturn(Optional.of(vt));
+	}
+
+	@Test
+	public void getCentersAroundCoords_GetsByRightCoords() {
+		var service = new VaccineCenterService(vaccineCenterRepository, vaccineTypeRepository);
```

Comment:
I'd say the usage of "var" is acceptable here, though, because we can clearly see the type of "service" from the right side of the assignment